js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNodeHealthRow.tsx

```diff
@@ -28,6 +28,8 @@ export const AssetNodeHealthRow = ({
   }, [health]);
 
   if (materializingRunId) {
+    const verb = definition.isMaterializable ? 'Materializing' : 'Observing';
```

Comment:
I think this is still not technically correct because assets can be both materialized and observed and the distinction is at the run level